media/js/firefox/australis/fx36/whatsnew.js

```diff
@@ -0,0 +1,53 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    var $tourStrings = $('#strings');
+    var helloAnimationStage = $('#hello-animation-stage');
+
+    //track if this is the first time a user has seen tour
+    function trackFirstTimeUse() {
+        var firstTime = 'True';
+        try {
+            if (localStorage.getItem('mozUITourGlobalFlag') === 'taken') {
+                firstTime = 'False';
+            } else {
+                localStorage.setItem('mozUITourGlobalFlag', 'taken');
+            }
+            gaTrack(['_trackEvent', 'Tour Interaction', 'First Time Seeing Tour', firstTime, 0, true]);
```

Comment:
> He also recommended keeping the First Time Seeing Tour event posting as is.

Ok, but just to be clear - the value for this will likely be `false` for most users (who have already seen the australis tour once). Is that data still useful?